docs/ecs_guide.md

```diff
@@ -0,0 +1,269 @@
+    
+![Launch](img/LaunchType_UI.png)
+<sub>Pictures of UI Cropped
+
+Next, set up an Prefect ECS worker that will discover and pull work from this work pool.
+## Step 2: Set up an ECS worker
+Start a Prefect worker in your ECS cluster.
+
+Create a Docker image for your Prefect worker, which you can build and push to the Amazon ECR registry. TK do you have to?
+
+For example, a minimal Dockerfile for an ECS worker could look like:
+```Dockerfile
+FROM prefecthq/prefect:2-python3.10
+RUN pip install s3fs prefect-aws
+```
+
+### Step 3: Create an IAM Role for the ECS task
+
+To create an IAM role for the ECS task using the AWS CLI, follow these steps: 
+
+1. **Create a Trust Policy**
+
+    The Trust Policy will specify that ECS can assume the role. 
+    
+    Save this policy to a file, such as `ecs-trust-policy.json`:
+
+    ```json
+
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "ecs-tasks.amazonaws.com"
+                },
+                "Action": "sts:AssumeRole"
+            }
+        ]
+    }
+    ```
+
+ 
+2. **Create the IAM Role**
+
+    Use the `aws iam create-role` command to create the role:
+
+    ```bash
+
+    aws iam create-role \ 
+    --role-name ecsTaskExecutionRole \
+    --assume-role-policy-document file://ecs-trust-policy.json
+    ```
+
+ 
+3. **Attach the Policy to the Role**
+
+    Amazon has a managed policy named `AmazonECSTaskExecutionRolePolicy` that grants the permissions necessary for ECS tasks. Attach this policy to your role:
+
+    ```bash
+
+    aws iam attach-role-policy \
+    --role-name ecsTaskExecutionRole \ 
+    --policy-arn arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
+    ```
+
+
+Remember to replace the `--role-name` and `--policy-arn` with the actual role name and policy Amazon Resource Name (ARN) you want to use.
+
+Now, you have a role named `ecsTaskExecutionRole` that you can assign to your ECS tasks. This role has the necessary permissions to pull container images and publish logs to CloudWatch.
+
+
+### Step 4: Create a task definition
+
+Next, create an ECS task definition that specifies the Docker image for the Prefect worker, the resources it requires, and the command it should run. In this example our command to run is `prefect worker start --pool my-ecs-pool`. 
+
+Here are the steps:
+
+1. Create a JSON file with the following contents:
+
+```json
+{
+  "family": "prefect-worker-task",
+  "networkMode": "awsvpc",
+  "taskRoleArn": "<your-ecs-task-role-arn>",
+  "executionRoleArn": "<your-ecs-task-role-arn>",
+  "containerDefinitions": [
+    {
+      "name": "prefect-worker",
+      "image": "<your-ecr-image>",
+      "cpu": 512,
+      "memory": 1024,
+      "essential": true,
+      "command": [
+        "prefect",
+        "worker",
+        "start",
+        "--pool",
+        "my-ecs-pool"
+      ],
```

Comment:
re my earlier comment this could be (i'd test it though!)

```suggestion
      "command": [
        "bash",
        "-c",
        "pip install prefect-aws && prefect worker start --pool my-ecs-pool",
      ],
```

Reply:
I've put:
```json
      "command": [
        "pip",
        "install",
        "prefect-aws",
        "&&",
        "prefect",
        "worker",
        "start",
        "--pool",
        "my-ecs-pool"
      ],
```

Reply:
You can't normally use `&&` in a infra command like that. You'd need to use `bash -c` first. Does this actually work?